cfpb/cfgov-refresh
# Adding button size capability to CTA and fixing issue with CTA inset logic

Adding button size capability to CTA and fixing issue with CTA inset logic


## Changes

- Modified code to add logic to display the CTA within an inset only when passed a flag.  The flag is only set from within the article macro. 
- Added a Button atom which inherits from the Hyperlink class. 
- Modified the macro to wrap the text in a div, which could be targeted.


## Testing

- Add a sub-landing page in Wagtail
- Add the Full-Width Text organism to the page
- Add the Call to Action to the page.
- Preview
- Edit the page and change the button size to large.
- Preview and verify the button is the correct size.

## Screenshots

- 
<img width="779" alt="screen shot 2017-04-19 at 2 33 28 pm" src="https://cloud.githubusercontent.com/assets/1696212/25196425/a2b85182-250e-11e7-808a-64a2356425f0.png">


## Notes

- The Jinja code is  regrettable and temporary, until we can resolve issues with the `render_stream_child` code. 

## Todos

-

## Checklist
